.github/workflows/action_tests.yaml

```diff
@@ -194,12 +194,12 @@ jobs:
         run: |
           cd repo-under-test
```

Comment:
should've mentioned this last time but: is it possible to do these assertions from src-repo?